src/Microsoft.IdentityModel.Logging/IdentityModelEventSource.cs

```diff
@@ -112,5 +141,17 @@ public static EventLevel LogLevel
                 _logLevel = value;
             }
         }
+
+        private string PrepareMessage(string message, EventLevel level)
+        {
+            if (message == null)
+            {
+                return message;
+            }
+
+            string currentTimeStamp = DateTime.UtcNow.ToString();
+            message = string.Format(CultureInfo.InvariantCulture, "[{0}]{1} {2}", level.ToString(), currentTimeStamp, message);
+            return message;
```

Comment:
I would recommend a slight coding change to remove local variables and string copies.

return string.Format(CultureInfo.InvariantCulture, "...", level.ToString(), DateTime.UtcNow, message);